actions/new.action.ts

```diff
@@ -203,7 +203,12 @@ const initializeGitRepository = async (dir: string) => {
 const createGitIgnoreFile = (dir: string, content?: string) => {
```

Comment:
```suggestion
const createGitIgnoreFile = async (dir: string, content?: string) => {
```

Reply:
Hi sorry for the dealy in response. Got stuck in my main project. I have no problem doing it this way, I was just trying to get the flow what was already there. If needed I can change.

Reply:
I believe async/await will be better to follow